src/util.rs

```diff
@@ -419,3 +419,165 @@ fn get_network_interfaces_impl() -> Vec<NetworkInterface> {
 
     vec
 }
+
```

Comment:
I'm gonna trust that you've calculated these checksums properly, and that they are correct - were they made up by either you or @david415, or were from examples in the spec?